src/lib/lib3index/cholesky.cc

```diff
@@ -1,12 +1,7 @@
 /*
- * @BEGIN LICENSE
+ *@BEGIN LICENSE
```

Comment:
Well, I am just puzzled by this.  So when I do a git log in cholesky_improvements, I see that my changes are a ahead of the last push to master (Daniel Smith's Fixes up DFT-D3 with modified -D3M.).  But, github is saying that my branch is far back as May 30th.